packages/react/src/components/FileUploader/FileUploader-test.js

```diff
@@ -293,6 +299,20 @@ describe('FileUploaderDropContainer', () => {
 
       expect(evt.target.value).toEqual(null);
     });
+
+    it('Fires onAddFiles upon selecting a file', () => {
+      const fileFoo = new File(['foo'], 'foo.txt', { type: 'text/plain' });
+      const fileBar = new File(['bar'], 'bar.txt', { type: 'text/plain' });
+      const mockFiles = [fileFoo, fileBar];
+      const input = mountWrapper.find(`.${prefix}--file-input`);
+      const evt = { target: { files: mockFiles } };
+      input.simulate('change', evt);
+      expect(onAddFiles).toHaveBeenCalledTimes(1);
+      expect(onAddFiles.mock.calls[0][0].target.files).toEqual([
```

Comment:
Maybe `expect(onAddFiles).toHaveBeenCalledWith` might be relevant here? Something like:

```js
expect(onAddFiles).toHaveBeenCalledWith(expect.objectContaining({
  target: {
    files: [fileFoo, fileBar],
  },
}));